proposals/cordova8.md

```diff
@@ -0,0 +1,53 @@
+We have come with a few ideas for the next major version of Cordova. 
+
+### Goals
+
+* Reduce maintenance burden of cordova tooling
+* Refactor cordova-lib 
+* Improve unit test coverage
+    
+### Proposed Changes
+
+* Remove non cordova-fetch fetching code
+    * remove no-fetch flag + any associated files with non cordova-fetch fetching
+    * drop npm dependency
+    * issue: https://issues.apache.org/jira/browse/CB-13055
+
+* Create new `cordova install` command
+    * remove restore functionality from `cordova prepare` command
+    * move restore functionality into new `cordova install` command. (installing plugins/platforms listed in `package.json` and `config.xml`)
+    * update docs to include `cordova install` 
+    * issue: https://issues.apache.org/jira/browse/CB-11981
+    * proposal: https://github.com/cordova/cordova-discuss/pull/54
+
+* Remove plugins dir from cordova projects
+This directory has traditionally been a staging directory. We copy plugins from the `plugins` directory to `platforms/platform/...` when a plugin needs to get installed. Now, these plugin directories also live in `node_modules` directory. I propose we delete the `plugins` directory and instead just copy the plugins straight from `node_modules`.
+    * TODO: investigate reason why `fetch.json`, + `<platform>.json` exist and if we need them. They live in plugins directory right now.
+    * https://issues.apache.org/jira/browse/CB-13059
+
+* Remove `cordova platform save` command
+This command isn't necessary anymore since we save by default since `cordova@7`. It was previously used to save added plugins and platforms to `config.xml`. 
```

Comment:
Looking at the analytics (assuming I'm doing this correctly), the command has been used 64 times out of the past 1.9 Million requests. Less that 0.1 percent. Pretty clear sign that it is not being used much.

I do see the usefulness of the command for users pre 7 who didn't used to use `--save` (which has been around much longer). I just don't think people in that group are even aware of the platform save command.

I think by this point asking users to modify config.xml or readd plugin + platforms is fair. They will probably have to readd/update older platforms or plugins anyways. 

Removing it will clean up cordova-lib a bit and allow us to get rid of these extra generated config files that go into peoples cordova projects